src/components/footer.tsx

```diff
@@ -72,8 +80,8 @@ const NigeriaFlag = (props) => (
   </chakra.svg>
 )
 
-export const Footer = () => (
-  <VStack as='footer' spacing={4} mt={12} textAlign='center'>
+export const Footer: React.FunctionComponent<StackProps> = (props) => (
```

Comment:
Why are we setting the `Footer` type to `React.FunctionalComponent` in this case?

Reply:
Looks like a relict from some trials I had. :D